src/osd/PGLog.h

```diff
@@ -1532,7 +1532,12 @@ struct PGLog : DoutPrefixProvider {
 	    OI_ATTR,
 	    bv);
 	  if (r >= 0) {
+#ifndef WITH_SEASTAR
 	    object_info_t oi(bv);
+#else
+	    object_info_t oi;
+	    oi.decode_without_oid(bv);
+#endif
```

Comment:
This pathway is also exclusive to classic.

Reply:
The crimson version is read_log_and_missing_crimson.

Reply:
dropped